consensus/src/simplex/actors/voter/actor.rs

```diff
@@ -648,7 +648,7 @@ impl<
         };
 
         // Request proposal from application
-        debug!(view = self.view, "requested proposal from automaton");
+        debug!(view = %self.view, "requested proposal from automaton");
```

Comment:
We use `?` elsewhere in the codebase? If this was auto-generated, add it to AGENTS.md

Reply:
The reason I used display was to avoid this looking like `view=View(10)`. The display implementation just prints the bare number.